closure/templates/test/BUILD

```diff
@@ -69,3 +71,16 @@ closure_js_test(
     ],
     language = "ECMASCRIPT6_STRICT",
 )
+
+closure_template_java_library(
```

Comment:
Oh actually could you change this to be `closure_java_template_library` since we renamed that literally just a few hours ago?